docs/CONTRIBUTING.md

```diff
@@ -97,13 +97,18 @@ need the following setup:
 <br/>
 
 ## Working with sources
-Please note that you should not use a code auto-formatter in this codebase.
+Please note that you should not use `cargo fmt` on this code base.
 Please read the following documents to learn more about reasons behind this
 decision and the recommended code style guide. Be sure to carefully read the
 [Rust style guide 1](./contributing/style-guide.md)
 and the [Rust style guide 2](https://dev.enso.org/docs/style-guide/rust.html)
 before contributing to the codebase.
 
+We do, however, use [`prettier`](https://prettier.io/) for the JavaScript files in our code base. 
+If you have not installed it already you can do so via `npm install prettier`. To use it manually 
+via command line run `prettier --write` to all JavaScript files in the project. Alternatively, 
+there are plugins for many IDEs available to do this for you.
```

Comment:
FYI, in my PR I've added `./run fmt` which auto formats all cargo toml files. We can extend it with auto formatting all js files too!